client/my-sites/plugins/plugin-autoupdate-toggle/index.jsx

```diff
@@ -52,10 +51,6 @@ module.exports = React.createClass( {
 			return null;
 		}
 
-		if ( ! this.props.wporg ) {
```

Comment:
I haven't tested the code but I don't think this looks right. 
A plugin can be a non .org plugin and still be visible in calypso. 

The `this.props.wporg` indicated if a plugin is found in the .org repo or not.